SConstruct

```diff
@@ -364,6 +368,10 @@ config_options = [
         "sphinx_docs",
         "Build HTML documentation for Cantera using Sphinx.",
         False),
+    BoolOption(
+        "clib_experimental",
+        "Build experimental CLib.",
+        False),
```

Comment:
Probably worth noting here that for this to work, you need to separately run both `scons doxygen` and `sourcegen` first.